game/resource/English/ability/units/boss/tooltip_as_passive_6.txt

```diff
@@ -0,0 +1,8 @@
+//=============================================================================
+// Simple Boss Desolate
+//=============================================================================
+"DOTA_Tooltip_ability_boss_as_passive_6"                      "Attack Speed Bonus"
+"DOTA_Tooltip_ability_boss_as_passive_6_Lore"                  "Roshan has been using this since 6.86."
+"DOTA_Tooltip_ability_boss_as_passive_6_Description"                "This unit's increased attack speed offsets its low base attack time. You can't slow its attacks as easily."
+"DOTA_Tooltip_ability_boss_as_passive_6_radius"            "RADIUS:"
```

Comment:
You don't need to have tooltips for every value if it's not really helpful. Saying "RADIUS: 0" will just look weird.